os/gtime/gtime_sql.go

```diff
@@ -26,3 +26,20 @@ func (t *Time) Value() (driver.Value, error) {
 	}
 	return t.Time, nil
 }
+
+// ValueForFieldType is the interface providing the Value method for package database/sql/driver
+// for retrieving value from golang variable to database.
+func (t *Time) ValueForFieldType(fieldType string) (driver.Value, error) {
```

Comment:
It is suggested adding `ValueForFieldType`  interface implement for `gtime.Time`, which adds unnecessary magic features for time object, could be complicated for usage.

Reply:
@cyjaysong

Reply:
> It is not suggested adding `ValueForFieldType` interface implement for `gtime.Time`, which adds unnecessary magic features for time object, could be complicated for usage.

@cyjaysong